.github/workflows/bin/spack-reviewers.py

```diff
@@ -0,0 +1,64 @@
+# Copyright Spack Project Developers. See COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import os
+import re
+import subprocess
+
+import requests
+
+IS_PACKAGE = re.compile(r"repos/spack_repo/builtin/packages/([^/]+)/package.py$")
```

Comment:
Not sure if we want/need to limit ourselves to package.py, if any files like patches are modified, we should notify the maintainers.

Reply:
Good idea, I've updated the code so we notify a maintainer if anything in the package's directory changes.